src/plugins/discord/fetcher.js

```diff
@@ -182,7 +182,10 @@ export class Fetcher implements DiscordApi {
       content: x.content,
       reactionEmoji: (x.reactions || []).map((r) => r.emoji),
       nonUserAuthor: x.webhook_id != null || false,
-      mentions: (x.mentions || []).map((user) => user.id),
+      mentions: (x.mentions || []).map((user) => ({
+        userId: user.id,
+        count: Array.from(x.content.matchAll(`<@!?${user.id}>`)).length,
```

Comment:
Question: Why do you need the question mark in the regex? Are there user IDs in the wild that don't have and `!` as part of the prefix?

Reply:
Yes, from the reference I linked:
> If they have a nickname there will also be a be a ! after the @.